test/DacpacTool.Tests/TestModelBuilder.cs

```diff
@@ -41,9 +41,15 @@ public TestModelBuilder AddStoredProcedureFromFile(string filename)
             return this;
         }
 
-        public TestModelBuilder AddReference(string path)
+        public TestModelBuilder AddReference(string path, string externalParts = "")
```

Comment:
"" => string.Empty

Reply:
`string.Empty` doesn't allow to use in parameters default.